source/_integrations/rachio.markdown

```diff
@@ -84,14 +84,51 @@ The `rachio` switch platform allows you to toggle zones and schedules connected
 
 Once configured, a switch will be added for every zone that is enabled on every controller in the account provided and a switch to start or stop every schedule on a controller. There will also be a switch to toggle each controller's standby mode, as well as to activate a 24 hour rain delay on the device.
 
-## Service
+## Services
 
-Rachio has one service that allows for setting the moisture percentage of a zone or group of zones. As Rachio only uses moisture levels for zones in a Flex Daily schedule, this service is only available when at least one zone is part of a Flex Daily schedule. 
+### Service `rachio.start_multiple_zone_schedule`
 
+Allows a list of zones to be passed with a corresponding list of durations to create a custom schedule directly from Home Assistant.
+
+<div class='note'>
+
+It is not currently possible to have zones from multiple controllers in the same custom schedule.
+
+</div>
+
+ Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id` | no | List of zones to run. Will be run in order listed.
+| `duration` | no | Duration in minutes to run the zones. If a list of durations is provided, each duration will apply to the corresponding zone listed above. Alternatively, one duration can be provided and will be used for all zones.
+
+### Examples:
+
+```yaml
+#Example sctipt to start multiple zones with individual duration for each zone.
+script:
+  run_grass_zones:
+    sequence: 
+      - service: rachio.start_multiple_zone_schedule
+        data:
+          entity_id: switch.front_yard_west, switch.front_yard_east, switch.side_yard_west
```

Comment:
We should avoid comma-separated strings as entity separators in the documentation.

```suggestion
          entity_id:
            - switch.front_yard_west
            - switch.front_yard_east
            - switch.side_yard_west
```